webapp/src/components/modals/channel_subscriptions/edit_channel_subscription.tsx

```diff
@@ -211,7 +219,29 @@ export default class EditChannelSubscription extends PureComponent<Props, State>
             }
         }
 
-        this.setState({filters, conflictingError: null});
+        const keys: string[] = [];
+        const statuses: Status[] = [];
+        if (this.state.projectStatuses) {
+            this.state.projectStatuses.forEach((element: ProjectStatuses) => {
```

Comment:
This variable name and type name are confusing me a bit. Is this an accurate understaning?

```suggestion
            this.state.projectStatuses.forEach((issueType: IssueTypeWithStatuses) => {
```